---
obj_interface/WorldObject[] objects